rotkehlchen/externalapis/cryptocompare.py

```diff
@@ -437,8 +437,9 @@ def query_current_price(
             self,
             from_asset: AssetWithOracles,
             to_asset: AssetWithOracles,
+            match_main_currency: bool,
             handling_special_case: bool = False,
-    ) -> Price:
+    ) -> Tuple[Price, bool]:
```

Comment:
you are changing the return type here and in many other function without adjusting the docstring. Adjust the docstring.